src/components/dropdownSearch/DropdownSearch.tsx

```diff
@@ -330,14 +331,20 @@ export class DropdownSearch extends React.Component<IDropdownSearchProps, void>
       width: this.props.width,
     };
 
-    return (
-      <div className={dropdownSearchClasses} style={dropdownSearchStyles}>
-        {this.getMainInput()}
+    const options: JSX.Element = !this.props.isOpened
+      ? null
```

Comment:
I agree that your suggestion is more readable, the logic here was simply to put the most common case first